### PR TITLE
chore: consolidate env for ci

### PR DESCRIPTION
**Pull request by brainbicycle**
The type of this PR is: **chore**

This PR resolves [MOPLAT-70]

### Description

Gets rid of old script to generate ci env file from circle vars and instead uses the same citadel env file for ci.

## Follow-ups

- [ ] Deploy some betas and make sure everything is working as expected
- [ ] Remove any redundant or unused vars from circle

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Use same citadel file for ci env vars - brian

</details>

[MOPLAT-70]: https://artsyproduct.atlassian.net/browse/MOPLAT-70?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 932c3b1b1b63c88f23c1672303e67eff9af094f0
</p>

File: scripts/setup-awscli-linux [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
```
Comment (pvinis) [inline scripts/setup-awscli-linux:3]: could we make these two files, one?

I'm pretty sure we have scripts that do that, and check for Darwin or Linux. then we can have 1 aws script, called the same from outside 🤔

**Review by pvinis — CHANGES_REQUESTED** [review]

File: scripts/setup-awscli-linux (hunk 1 above)
Comment (brainbicycle) [reply scripts/setup-awscli-linux:3]: I don't actually see us doing that anywhere but it is a nice idea

**Comment by MounirDhahri** [comment]
LGTM, I will as well trigger some betas from your branch to make sure everything works fine

**Review by MounirDhahri — DISMISSED** [review]
Worked like a charm 🚀

File: scripts/setup-awscli-linux (hunk 1 above)
Comment (brainbicycle) [reply scripts/setup-awscli-linux]: alright I did this here and in the setup distribute script!

File: src/lib/ErrorReporting.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ export const setupSentry = (
   props: Partial<Sentry.ReactNativeOptions> = {},
   captureExceptions = !__DEV__
 ) => {
-  if (captureExceptions && Config.SENTRY_DSN) {
+  if (!captureExceptions && Config.SENTRY_DSN) {
```
Comment (brainbicycle) [inline src/lib/ErrorReporting.ts:11]: this was supposed to be on another branch, although I am still confused how this is working at all

**Review by pvinis — APPROVED** [review]

**Comment by pvinis** [comment]
lets go!

**Pull request merged**